web/src/services/thirdparty.js

```diff
@@ -21,7 +20,6 @@ class ConnectionsService {
     this.connections = [];
     if (isProduction()) {
       this.addConnection(new twitterConnection());
-      this.addConnection(new mixpanelConnection());
```

Comment:
We should also remove mixpanel stuff from index.ejs :)

Reply:
I don't think there is anything? I've tried searching for things from the docs

https://developer.mixpanel.com/docs/javascript-quickstart

Reply:
Oh you're right, I don't know where I saw this 🤔